test/partiql-tests-runner/src/main/kotlin/org/partiql/runner/ConformanceComparison.kt

```diff
@@ -4,98 +4,223 @@ import com.amazon.ionelement.api.loadSingleElement
+| --- | ---: | ---: | ---: |
+| % Passing | ${"%.2f".format(firstPassingPercent)}% | ${"%.2f".format(secondPassingPercent)}% | ${"%.2f".format(secondPassingPercent - firstPassingPercent)}% |
+| :white_check_mark: Passing | $firstPassingSize | $secondPassingSize | ${secondPassingSize - firstPassingSize} |
+| :x: Failing | $firstFailingSize | $secondFailingSize | ${secondFailingSize - firstFailingSize} |
+| :large_orange_diamond: Ignored | $firstIgnoreSize | $secondIgnoreSize | ${secondIgnoreSize - firstIgnoreSize} |
+| Total Tests | $firstTotalSize | $secondTotalSize | ${secondTotalSize - firstTotalSize} |
+                """.trimIndent()
+            )
+            this.appendLine(
+                """
+Number passing in both: ${passingInBoth.count()}
 
-    if (!passingOrigFailingNew.isEmpty()) {
-        comparisonReportFile.appendText(
-            "\n:interrobang: CONFORMANCE REPORT REGRESSION DETECTED :interrobang:. The following test(s) were previously passing but now fail:\n<details><summary>Click here to see</summary>\n\n"
-        )
-        passingOrigFailingNew.forEach { testName ->
-            comparisonReportFile
-                .appendText("- ${testName}\n")
-        }
-        comparisonReportFile
-            .appendText("\n</details>")
-    }
+Number failing in both: ${failingInBoth.count()}
 
-    if (!failureOrigPassingNew.isEmpty()) {
-        comparisonReportFile.appendText(
-            "\nThe following test(s) were previously failing but now pass. Before merging, confirm they are intended to pass: \n<details><summary>Click here to see</summary>\n\n"
-        )
-        failureOrigPassingNew.forEach { testName ->
-            comparisonReportFile
-                .appendText("- ${testName}\n")
+Number passing in ${first.engine} engine but fail in ${second.engine} engine: ${passingFirstFailingSecond.count()}
+
+Number failing in ${first.engine} engine but pass in ${second.engine} engine: ${failureFirstPassingSecond.count()}
+                """.trimIndent()
+            )
+            if (passingFirstFailingSecond.isNotEmpty()) {
+                if (passingFirstFailingSecond.size < limit) {
+                    this.appendLine(":interrobang: CONFORMANCE REPORT REGRESSION DETECTED :interrobang:. The following test(s) are passing in ${first.engine} but fail in ${second.engine}:\n<details><summary>Click here to see</summary>\n\n")
+
+                    passingFirstFailingSecond.forEach { testName ->
+                        this.appendLine("- $testName")
+                    }
+                    this.appendLine("</details>")
+                } else {
+                    this.appendLine(":interrobang: CONFORMANCE REPORT REGRESSION DETECTED :interrobang:")
+                    this.appendLine("Download Artifact from Summary to view the complete list")
+                }
+            }
+
+            if (failureFirstPassingSecond.isNotEmpty()) {
+                if (failureFirstPassingSecond.size < 10) {
```

Comment:
Should be `... < limit`?

```suggestion
                if (failureFirstPassingSecond.size < limit) {
```